apm-server/README.md

```diff
@@ -8,7 +8,8 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
-**Warning**: This branch is used for development, please use the latest [7.x][] release for released version.
+<!-- development warning -->**Warning**: This branch is used for development, please use the latest [7.x][] release for released version.
```

Comment:
```
<!-- development warning placeholder -->
**Warning**: This branch is used for development, please use the latest [7.x][] release for released version.
```

I have 2 suggestions:
- maybe add "placeholder" in the markdown comment to make it more explicit what it is, especially when the warning is removed in released version
- maybe move placeholder comment and warning statements on different lines to fix the markdown format issue